index.html

```diff
@@ -0,0 +1,125 @@
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+  <title>Apple-project</title>
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <section class="logo container">
+    <nav>
+      <ul class="flex nav text">
+        <li class="links">
+          <a href="#"><i class="fa fa-apple"></i></a>
+        </li>
+        <li class="links item">
+          <a href="#">Store</a>
+        </li>
+        <li class="links item">
+          <a href="#">Mac</a>
+        </li>
+        <li class="links item">
+          <a href="#">iPod</a>
+        </li>
+        <li class="links item">
+          <a href="#">iPhone</a>
+        </li>
+        <li class="links item">
+          <a href="#">iPad</a>
+        </li>
+        <li class="links item">
+          <a href="#">iTunes</a>
+        </li>
+        <li class="links item">
+          <a href="#">Support</a>
+        </li>
+        <li class="links">
+          <form>
+            <input type="text">
+            <span><i class="fa fa-search"></i></span>
+          </form>
+        </li>
+      </ul>
+    </nav>
+    <section class="main text">
+      <div>
+        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/">
+          <img src="images/your_verse_title_large.png" alt="Verse">
+          <img class="push-up" src="images/your_verse_subhead_small.png" alt="your-verse">
+        </a>
+        <br>
+        <a class="white flex" href="#">
+          <h3 class="h1">
+            Watch the "Your Verse" film and explore the stories
+            &ensp;
+            <i class="fa fa-play-circle" aria-hidden="true"></i>
+          </h3>
+        </a>
+      </div>
+    </section>
+    <article class="gallery flex">
+      <a href="#"><img src="images/promo_iphone5c.jpg" alt="gallery image iPhone"></a>
+      <a href="#"><img src="images/promo_iphone5s.jpg" alt="gallery image iPhone"></a>
+      <a href="#"><img src="images/promo_30_years.jpg" alt="gallery image years"></a>
+      <a href="#"><img src="images/promo_supplier_responsibility.jpg" alt="supplier image"></a>
+    </article>
+  </section>
+  <footer class="main">
+    <div class="section-one">
+      <p>Shop the
+        <a href="https://web.archive.org/web/20140301004610/http://store.apple.com/us">Apple Online Store</a>
+        (1-800-MY-APPLE), visit an <a
+          href="https://web.archive.org/web/20140301004610/http://www.apple.com/retail/">Apple Retail Store</a> , or
+        find <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/buy/">reseller</a></p>
+        <div class="section-three">
+          
+           <p> <small>Copyright © 2014 Apple Inc. All rights reserved.<a class="terms" href="https://web.archive.org/web/20140301004610/http://www.apple.com/legal/">Terms of Use</a><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/legal/privacy/">Privacy Policy</a>
+          </small>
```

Comment:
- [ ] Please move this section a little upper so that it can look like the original. It is good practice to always have a margin at the bottom of the page
![image](https://user-images.githubusercontent.com/53256139/88931820-f21eca00-d242-11ea-8f75-1dd75a2d5042.png)

Reply:
done